addon/components/tooltip-on-parent.js

```diff
@@ -25,7 +25,9 @@ export default Ember.Component.extend({
         console.warn('No renderTooltip method found on the parent view of the {{tooltip-on-parent}} component');
       }
 
-      this.remove();
+      if (!this.get('isDestroyed') && !this.get('isDestroying')) {
```

Comment:
Do we need `isDestroyed`? Isn't `isDestroying` true when `isDestroyed` is true? Looking at the [Ember docs](http://emberjs.com/api/classes/Ember.CoreObject.html#property_isDestroying) I'm not sure. Let me know how you want to proceed and we can get this merged.